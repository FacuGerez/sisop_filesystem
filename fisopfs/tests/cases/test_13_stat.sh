#!/bin/bash

MOUNT=tests/mount

touch "$MOUNT"/testfile
stat "$MOUNT"/testfile > "$MOUNT"/statfile
cat "$MOUNT"/statfile | sed 's/^[[:space:]]*//' | cut -d ':' -f1