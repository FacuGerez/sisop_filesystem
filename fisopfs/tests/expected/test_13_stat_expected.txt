File
Size
Device
Access
Access
Modify
Change
Birth
